cdap-ui/app/cdap/components/HttpExecutor/RequestHistoryTab/index.tsx

```diff
@@ -102,9 +112,20 @@ const styles = (theme): StyleRules => {
   };
 };
 
+const StyledExpansionPanel = withStyles(() => ({
+  root: {
+    '&$expanded': {
+      margin: 0,
+    },
+    borderBottom: '1px solid #C0C0C0',
```

Comment:
use color already defined in the theme palettes

Reply:
Done